azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/bindings/Binding.java

```diff
@@ -100,4 +100,21 @@ public String toString() {
                 .append(" ]")
                 .toString();
     }
+
+    protected String getDirectionFromAttribute() {
+        if (this.direction != null) {
```

Comment:
We may split this function, do the null check of direction in `getDirection` and only get direction from attribute map here.